app/src/main/java/com/cornellappdev/transit/ui/components/MenuItem.kt

```diff
@@ -0,0 +1,34 @@
+package com.cornellappdev.transit.ui.components
+
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun MenuItem (label: String, sublabel: String) {
+    Row(Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically){
+        Icon(Icons.Filled.Place, contentDescription = "Place", modifier = Modifier.padding(horizontal = 10.dp))
+        Column () {
+            Text(text = label, fontSize = 24.sp)
+            Text(text = sublabel, fontSize = 16.sp)
+        }
+    }
+}
```

Comment:
🚨🚨🚨 SINGULAR OFF SPACING DETECTED 🚨🚨🚨 
(make sure to lint the codebase before pushing c:)

Also, isn't really a massive deal for usage, but just for future readability could you add documentation comments to composables as well just explaining what they are?